README.md

```diff
@@ -149,6 +42,14 @@ sudo su -
 source ./sample_wcp_envs.sh
 ./install-wcp.bash
 ```
+### Sample Helper Scripts Usage
+In addition to the sample installation environment files. This repo includes a few other sample environment files to 
+assist in various configurations and use cases.  The power and flexibility of the script's modular functions allow for 
+varied use cases such as installing java, installing java & tomcat updating tomcat etc. We have included a few examples 
```

Comment:
missing commas: installing java & tomcat, updating tomcat, etc.

Reply:
edited, thank you.